tsa/tsacmd/command.go

```diff
@@ -141,6 +144,46 @@ func (cmd *TSACommand) Runner(args []string) (ifrit.Runner, error) {
 		httpClient:        httpClient,
 		sessionTeam:       sessionAuthTeam,
 	}
+	// Starts a goroutine that his purpose is to basically listen to the
+	// SIGHUP syscall to then reload the config.
+	// For now it only reload the TSACommand.AuthorizedKeys but any
+	// other configuration could be added
+	go func() {
+		// Set up channel on which to send signal notifications.
+		// We must use a buffered channel or risk missing the signal
+		// if we're not ready to receive when the signal is sent.
+		c := make(chan os.Signal, 1)
+		defer close(c)
+		signal.Notify(c, syscall.SIGHUP)
+		for {
+
+			// Block until a signal is received.
+			_ = <-c
```

Comment:
I believe the idiomatic way doesn't require the blank identifier. I would change it to keep it consistent with how blocking is done [elsewhere](https://github.com/concourse/concourse/blob/c4261d94405c3a7cc200b7ed8bd3f0ed84b12b4b/worker/volume_sweeper.go#L92) in the code.
```suggestion
			<-c
```